apps/web/components/Gallery/Gallery.vue

```diff
@@ -180,6 +180,22 @@ const registerThumbsWatch = (
 registerThumbsWatch(firstThumbReference, firstVisibleThumbnailIntersected);
 registerThumbsWatch(lastThumbReference, lastVisibleThumbnailIntersected);
 
+const getWidth = (image: ImagesData, imageUrl: string) => {
+  const imageWidth = productImageGetters.getImageWidth(image) || 600;
+  if (imageWidth && imageWidth > 0 && imageUrl.includes('/full/')) {
+    return imageWidth;
+  }
+  return '';
+};
+
+const getHeight = (image: ImagesData, imageUrl: string) => {
+  const imageHeight = productImageGetters.getImageHeight(image) || 600;
+  if (imageHeight && imageHeight > 0 && imageUrl.includes('/full/')) {
```

Comment:
If the full image url changes we are left with changing a hard-coded string in multiple places. Please consider making this a general constant. We have `defaults.ts` file for these.